src/commands/deleteContainerizedFunctionApp/DeleteFunctionAppWizardContext.ts

```diff
@@ -0,0 +1,13 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { type Site } from "@azure/arm-appservice";
+import { type ExecuteActivityContext, type IActionContext } from "@microsoft/vscode-azext-utils";
+import { type ContainerTreeItem } from "../../tree/containerizedFunctionApp/ContainerTreeItem";
+
+export interface DeleteFunctionappWizardContext extends IActionContext, ExecuteActivityContext {
+    site: Site;
+    proxyTree: ContainerTreeItem;
```

Comment:
Actually one other thing, is the only reason we have the proxyTree on the context for the subscription?  If so, maybe you can just build an `ISubscriptionActionContext` at the beginning so you don't have to store the whole tree item reference on the context.

Reply:
I don't think I can do this since the I don't have access to the `AzureSubscription` either but I made some changes and put the subscription in the constructor so I could reuse it. Let me know if that's better/worse.